config/jest/SetupTestEnvironment.js

```diff
@@ -17,6 +17,9 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
+
+require('whatwg-fetch');
```

Comment:
Just so that I understand, it is some sort of polyfill that implements Reponse because it is missing in the test env ?

Reply:
Exactly